src/test/java/org/folio/rest/impl/RestVerticleTest.java

```diff
@@ -19,68 +24,167 @@
 
 import java.io.IOException;
 
+import static org.folio.rest.impl.ApiTestHelper.doRequest;
+
 @RunWith(VertxUnitRunner.class)
 public class RestVerticleTest {
 
   private Vertx vertx;
+  private final Logger logger = LoggerFactory.getLogger("TemplateEngineTest");
+  private static String templateUrl;
+  private static String okapiUrl;
+
+  private JsonArray outputFormat = new JsonArray()
+    .add("text")
+    .add("html");
+
+  private JsonObject template1 = new JsonObject()
+    .put("lang", "en")
+    .put("template", "Hello ${context.user.name}");
+
+  private JsonObject template2 = new JsonObject()
+    .put("lang", "de")
+    .put("template", "Hallo ${context.user.name}");
+
```

Comment:
Would it be better to have generalized method like this?
private JsonObject buildLocalizedTemplate(String locale){
JsonObject template = new JsonObject()
    template.put("lang", locale)
    template.put("template", "Hallo ${context.user.name}");
return template;
}

Reply:
there are two different templates too. So create a no-reuse method with two params for two objects doesn't looks like needed here